fe/fe-core/src/main/java/com/starrocks/sql/ast/AddMaterializedColumnClause.java

```diff
@@ -0,0 +1,45 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
```

Comment:
What expressions do you provide for the materialized column? Does the SchemaChangeJobV2 in BE support the get_json_string?